src/test/groovy/unit/org/pih/warehouse/inventory/CycleCountItemSpec.groovy

```diff
@@ -0,0 +1,78 @@
+package unit.org.pih.warehouse.inventory
+
+import grails.testing.gorm.DomainUnitTest
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.inventory.CycleCountItem
+import org.pih.warehouse.inventory.InventoryItem
+import spock.lang.Specification
+
+class CycleCountItemSpec extends Specification implements DomainUnitTest<CycleCountItem> {
+
+    void 'getQuantityVariance should return #quantityVariance when quantityCounted is #quantityCounted and qoh is #qoh'() {
+        given:
+        domain.quantityCounted = quantityCounted
+        domain.quantityOnHand = quantityOnHand
+
+        expect:
+        domain.quantityVariance == quantityVariance
+
+        where:
+        quantityCounted | quantityOnHand || quantityVariance
+        2               | 2              || 0
+        10              | 3              || 7
+        10              | 20             || -10
+        null            | 1              || null
+        1               | null           || null
+    }
+
+    void 'compareTo should return items in order starting with expiration date and location name'() {
+        given:
+        mockDomain(InventoryItem)
+        mockDomain(Location)
```

Comment:
you can do `mockDomains(InventoryItem, Location)`. But do you even need to mock the domains? I usually don't need to unless I call `.save()` on the domain